ynr/apps/moderation_queue/review_required_helper.py

```diff
@@ -224,15 +225,14 @@ def needs_review(self):
         from candidates.models import LoggedAction
 
         if self.logged_action.action_type == "person-revert":
-            update_qs = LoggedAction.objects.filter(
-                person=self.logged_action.person, action_type="person-update"
+            recent_revert_qs = LoggedAction.objects.filter(
+                person=self.logged_action.person,
+                action_type="person-revert",
+                # updated in the last 24 hours
+                updated__gt=datetime.now() - timedelta(hours=24),
             ).order_by("updated")
-            # filter out edits that have the current user as the user
-            update_qs = update_qs.exclude(user=self.logged_action.user)
-            # find the last edit made by another user
-            if update_qs.exists():
-                if self.logged_action.user != update_qs.last().user:
-                    return self.Status.NEEDS_REVIEW
+            if recent_revert_qs.count() >= 2:
+                return self.Status.NEEDS_REVIEW
```

Comment:
@symroe I hope this is what you were after. Let me know and if so, I can update tests to match.

Reply:
Yep, I think this deals with most of the issues...we might want a more advanced version in time if this doesn't work, but I think this is easier to understand at lease!